src/components/member.js

```diff
@@ -0,0 +1,30 @@
+import { members } from '../data.js';
+
+export default function Member() {
+    const membersList = members.map(member =>
+        <div className="FlexDisplay">
```

Comment:
you should pass a `key` for every element in map

Reply:
for all tags or for all members?

Reply:
No just the element in the top level. Consider this resolved.